### ISSUE TITLE
Allow NX 15

### ISSUE THREAD

**Issue by ianldgs** [gperdomor/nx-tools#624]
NX just released version 15. It has only one breaking change.

`@nx-tools/nx-docker` and `@nx-tools/docker-metadata` don't allow NX 15:

```
npm ERR! While resolving: frontend@0.0.0
npm ERR! Found: @nrwl/devkit@15.0.0
npm ERR! node_modules/@nrwl/devkit
npm ERR!   dev @nrwl/devkit@"15.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @nrwl/devkit@"^12.0.0 || ^13.0.0 || ^14.0.0" from @nx-tools/docker-metadata@3.0.1
npm ERR! node_modules/@nx-tools/docker-metadata
npm ERR!   dev @nx-tools/docker-metadata@"^3.0.1" from the root project
```

**Comment by gperdomor** [comment]
@ianldgs try with docker-metadata `3.0.2` and nx-docker `3.0.5`

**Comment by chromey** [comment]
The problem seems to be the dependency on `@nx-tools/core` which has no released version that supports nx 15.

```
├─┬ @nx-tools/nx-docker@3.0.5
│ ├─┬ @nx-tools/core@3.0.2
│ │ └── @nrwl/devkit@14.8.6
```

**Comment by gperdomor** [comment]
closed with version 3.0.7... All packages are in sync now... :D...
